siliconcompiler/tools/openroad/cts.py

```diff
@@ -0,0 +1,19 @@
+
+from .openroad import setup as setup_tool
+
+def setup(chip):
+    ''' Helper method for configs specific to cts tasks.
+    '''
+
+    # Generic tool setup.
+    setup_tool(chip)
+
+    tool = 'openroad'
+    task = 'cts'
+    design = chip.top()
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+
+    if (not chip.valid('input', 'layout', 'def') or
+        not chip.get('input', 'layout', 'def')):
```

Comment:
not needed, only floorplan can take in a def this way at the moment (all but floorplan)